components/brave_wallet_ui/common/trezor/trezor-bridge-transport.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import { kTrezorBridgeUrl, MessagingTransport, TrezorFrameCommand } from './trezor-messages'
+
+// Handles sending messages to the Trezor library, creates untrusted iframe,
+// loads library and allows to send commands to the library and subscribe
+// for responses.
+export class TrezorBridgeTransport extends MessagingTransport {
+  constructor (bridgeFrameUrl: string) {
+    super()
+    this.bridgeFrameUrl = bridgeFrameUrl
+    // @ts-ignore
+    this.frameId = crypto.randomUUID()
+  }
+
+  private frameId: string
+  private bridgeFrameUrl: string
+
+  // T is response type, e.g. UnlockResponse. Resolves as `false` if transport error
+  sendCommandToTrezorFrame = async<T> (command: TrezorFrameCommand): Promise<T | false> => {
```

Comment:
nit: this doesn't need to be `async` since it doesn't do any `await` (that happens inside another `async` function.

Reply:
done